docs/extensions/datacollector-usage.md

```diff
@@ -0,0 +1,88 @@
+# Using DataCollectors
```

Comment:
Please merge this doc with `analyze.md`, the earlier doc specifically is about Data Collectors.